### ISSUE TITLE
Log and display requests/hits when a tunnel is running

### ISSUE THREAD

**Issue by matiassingers** [localtunnel/localtunnel#41]
Would be super cool with a verbose mode that displayed all requests to the tunnel currently running.

Kind of like `heroku logs` etc. does it:

``` bash
$ lt --port 4000
your url is: https://yxyrzgfrbx.localtunnel.me

2014-04-14T10:10:06.747431+00:00 method=GET path=/ connect=1ms service=37ms status=200 bytes=290
```

**Comment by rayshan** [comment]
+1

**Comment by nguyenvanhau** [comment]
+1

**Comment by ngelx** [comment]
+1

**Comment by petersandor** [comment]
+1

**Comment by Pagebakers** [comment]
+1

**Comment by alvar21** [comment]
+1

**Comment by mateusmaso** [comment]
+1

**Comment by Mangor** [comment]
+1

**Comment by ganglio** [comment]
+1

**Comment by jonchurch** [comment]
+1

**Comment by goranefbl** [comment]
+1

**Comment by defunctzombie** [comment]
Due to a recent refactor of how requests are proxied, I think this could not be easier to implement if someone wants to take a stab at it. I don't be adding this feature myself anytime soon.

**Comment by dmk255** [comment]
I've made a light weight PR to add some basic logging.

It only returns a timestamp, method, and path but that should be generally useful for most cases.

**Comment by defunctzombie** [comment]
Merged #178; after some other client library cleanup I will release a new version.
